launcher/updater.cpp

```diff
@@ -493,12 +493,15 @@ void Updater::taskDownload(QDir &installDir, const QUrl &url, const QString &has
         // (Yeah, we're gonna do this crappy trick again...)
         eventLoop.reset(new QEventLoop());
         QArchive::DiskExtractor extractor(filename, installDir.path(), this, false);
+        // Note. The parmeters of the signal QArchive::DiskExtractor::error do not match the slot.
+        // QArchive::DiskExtractor::error emits short code only
         QObject::connect(&extractor, &QArchive::DiskExtractor::error,
-                         [&](short code, const QString &file) {
-            error = true;
-            errorMsg = file;
-            errorCode = code;
-            qCritical() << "Error extracting" << file << "- code" << code;
+                         [&](short code) {
+             const QString file("Unknown file");
+             error = true;
+             errorMsg = file;
+             errorCode = code;
+             qCritical() << "Error extracting" << file << "- code" << code;
```

Comment:
The change in this signal's signature is due to a major version change in QArchive that apparently was applied to this branch. I would be careful because I have not tested that new version.